index.js

```diff
@@ -8,34 +8,26 @@
 //4) ignores non-numeric values  and continues operation
 //5) returns NaN if the array does not represent a proper operation
 
-// function calculate (HazyCalculator)
-const HazyCalculator = (ArrayNumbers, Operands) => {
-//convert a string to a number
-parseInt ('1', 1)
-parseInt ('2', 2)
-parseInt ('3', 3)
-parseInt ('4', 4)
-parseInt ('5', 5)
-parseInt ('6', 6)
-parseInt ('7', 7)
-parseInt ('8', 8)
-parseInt ('9', 9)
-// multiplies two numbers when one is stringified
-var ArrayNumbers = 
-A = 2
-B = "2"
-Return (A) * number(B)
-// Convert Null values as zeros
-const swapValue = (HazyCalculator) => {
-    Object.keys(HazyCalculator).forEach(key => {
-       if(!obj[key]){
-          obj[key] = '0';
-// Undefined values ignored
-var ArrayNumbers = [, , '', 23, '-', , 12];
-ArrayNumbers = ArrayNumbers.filter(function( element ) {
-   return element !== undefined;
-});
-// Empty string values ignored
-// Non-numeric values ingored
+//function calculate (CalculationSteps)
+//* eslint-disable comma-spacing */
+/* eslint-disable no-sparse-arrays */
```

Comment:
when you disable your eslint you miss mistakes. Don't disable, FIX
![image](https://user-images.githubusercontent.com/71082001/114576529-3921e380-9c49-11eb-8f10-9de85d5c73aa.png)